src/colors.less

```diff
@@ -1,7 +1,11 @@
 @primary-bg: rgba(0, 0, 0, 0.85);
 @secondary-bg: rgba(255, 255, 255, 0.065);
+@tertiary-bg: rgba(255, 255, 255, 0.3);
 
 @primary-shadow: rgba(0, 0, 0, 0.08);
 
+
 @primary: rgb(255, 255, 255);
-@accent: rgb(145, 49, 37);
+@accent: #F28123;
```

Comment:
Do we somewhere have a definition of the "Rembrandt colors"?

Reply:
I will put together a coloors collection.

Reply:
https://coolors.co/252525-252525-343434-f28122-ffffff

Reply:
:+1: Added it to the drive